README.md

```diff
@@ -1,79 +1,99 @@
+## ⚡️ Comparison with Twitch Launcher
 
-GDLauncher is a custom open source Minecraft launcher written from the ground up in electron/react. Its main goal is to make it easy and enjoyable to manage different Minecraft versions and servers, bringing the playing and modding experience to the next level!
+This is an example of the time that GDLauncher takes to install a modpack in comparison to Twitch. Both tests are running at the same time over a 1Gbps network to ensure that the network doesn't impact the comparison.
+- GDLauncher: `0:52m`
+- Twitch Launcher: `2.25m`
+
+<p align="center">
+    <img width="800" height="auto" src="https://gdevs.io/comparison.gif" alt="GDLauncher" />
+</p>
 
 ## 🚀 Getting Started
 
-Below you will find everything you need to know about the launcher. If you want to download a stable release you can do it from our official website (https://gdevs.io). If you want the latest testing features you can clone the repo and compile it yourself.
+Below you will find everything you need to know about the launcher. If you want to download the latest stable release you can do it from our official website ([gdevs.io](https://gdevs.io)). If you want to test the possibly unstable features, you can clone the repo and compile it yourself.
 
 ## 🎮 Download
 
 To download the latest version, you can either click [here](https://github.com/gorilla-devs/GDLauncher/releases) and select the appropriate version for your operating system, or visit our [website](https://gdevs.io).
 
-## 🎨 Features (Working on)
+## 🎨 Features
 
-- Java Autorunner. You don't need to have java installed, a suitable version will be downloaded automatically.
-- Log console. Always know what's happening.
-- Easy installation. It's as easy as pie to install both the vanilla game and forge. No further action from the user is required.
-- Built-in autoupdater. You will never need to download a new version manually.
-- Vanilla, Forge, and Curse modpacks download and autoupdater.
-- Built-in manager for Minecraft servers.
-- Cloud sync of game saves. You will never lose your saves again!
+#### Our features:
 
-You can also:
+- Java downloader. You don't need to have java installed, a suitable version will be downloaded automatically.
+- It's as easy as pie to install the `vanilla` game, `forge`, `fabric` and all `twitch modpacks`. No further action from the user is required.
+- Install `mods` for both fabric and forge directly from our UI
+- Built-in autoupdater. The launcher will always keep itself updated to the latest release.
+- Easily manage multiple `accounts` and switch between them.
+- Still playing on your grandma pc from the 80s? Don't worry, we got you covered with our `Potato PC Mode`!
 
-- Import and export modpacks from and to other launchers
-- Drag and drop instances wherever you like them, just like in your desktop
+#### You can also:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5447851)